app/src/main/java/com/example/xpvehicles/adapters/RentingRequestsAdapter.java

```diff
@@ -106,36 +87,50 @@ private void bind(View itemView) {
             tvRentingPickUpDate = itemView.findViewById(R.id.tvRentingPickUpDate);
             tvRentingReturnDate = itemView.findViewById(R.id.tvRentingReturnDate);
-            ivRentingVehicleImage = itemView.findViewById(R.id.ivRentingVehicleImage);
             tvStatus = itemView.findViewById(R.id.tvStatus);
+            viewPager = itemView.findViewById(R.id.viewPagerRentingVehicleImages);
+            tabLayout =  itemView.findViewById(R.id.tabLayout);
         }
 
-        public void setValues(RentVehicle rentvehicle) throws ParseException{
+        public void setValues(RentVehicle rentVehicle) throws ParseException{
             // Vehicle name
-            Vehicle originalVehicle = (Vehicle) rentvehicle.getVehicle();
+            Vehicle originalVehicle = (Vehicle) rentVehicle.getVehicle();
             String vehicleName = originalVehicle.fetchIfNeeded().getString("name");
             tvRentingVehicleName.setText(vehicleName);
 
             SimpleDateFormat sdf = new SimpleDateFormat("MM/dd/yyyy");
             // vehicle pickup date
-            Date pickUpDate = rentvehicle.getPickUpDate();
+            Date pickUpDate = rentVehicle.getPickUpDate();
             String formattedPickUpDate = sdf.format(pickUpDate);
             tvRentingPickUpDate.setText(formattedPickUpDate);
             // vehicle return date
-            Date returnDate = rentvehicle.getReturnDate();
+            Date returnDate = rentVehicle.getReturnDate();
             String formattedReturnDate = sdf.format(returnDate);
             tvRentingReturnDate.setText(formattedReturnDate);
 
             // status
-            String status = rentvehicle.getStatus();
+            String status = rentVehicle.getStatus();
             tvStatus.setText(status);
             setStatusColor(status);
 
             // vehicle image
-            ParseFile image = originalVehicle.getVehicleImage();
-            if (image != null) {
-                Glide.with(fragment.getContext()).load(image.getUrl()).into(ivRentingVehicleImage);
-            }
+            bindVehicleImagesAdapter(originalVehicle);
+        }
+
+        private void bindVehicleImagesAdapter(Vehicle rentVehicle) {
+            List<ParseFile> images = rentVehicle.getVehicleImages();
+            VehicleImagesAdapter vehicleImagesAdapter =  new VehicleImagesAdapter(activity, images);
+            viewPager.setAdapter(vehicleImagesAdapter);
+
+            //indicator dots at the bottom
+            TabLayoutMediator tabLayoutMediator =
+                    new TabLayoutMediator(tabLayout, viewPager, true,
+                            new TabLayoutMediator.TabConfigurationStrategy() {
+                                @Override public void onConfigureTab(
+                                        @NonNull TabLayout.Tab tab, int position) { }
+                            }
+                    );
+            tabLayoutMediator.attach();
```

Comment:
I believe that this block of code is duplicate. Let's refactor into some module that can be used by multiple adatpers: ex: inheritance